docs/ops/arithmetic/FloorMod_1.md

```diff
@@ -4,41 +4,49 @@
 
 **Category**: Arithmetic binary operation
 
-**Short description**: *FloorMod* returns an element-wise division reminder with two given tensors applying multi-directional broadcast rules. 
-The result here is consistent with a flooring divide (like in Python programming language): `floor(x / y) * y + mod(x, y) = x`. 
-The sign of the result is equal to a sign of the divisor.
+**Short description**: *FloorMod* performs an element-wise floor modulo operation with two given tensors applying broadcasting rule specified in the *auto_broadcast* attribute. 
+
+**Detailed description**
+As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *FloorMod* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
+
+\f[
+o_{i} = a_{i} % b_{i}
+\f] 
+
+*FloorMod* operation computes a reminder of a floored division. It is the same behaviour like in Python programming language: `floor(x / y) * y + floor_mod(x, y) = x`. The sign of the result is equal to a sign of a dividend.
```

Comment:
May be it make sense to add a comment for case when the divisor is 0 that the output value is undefined.